bool ampere_feedback
int32 ampere_threshold
---
bool set_ok